client/modules/IDE/components/Preferences.jsx

```diff
@@ -66,238 +72,234 @@ class Preferences extends React.Component {
+                onClick={() => {
+                  this.indentationInput.select();
+                }}
+              >
+              </input>
+              <button
+                className="preference__plus-button"
+                onClick={() => this.props.setIndentation(this.props.indentationAmount + 2)}
+                aria-label="increase indentation amount"
+              >
+                <InlineSVG src={plusUrl} alt="IncreaseIndentation Amount" />
+                <h6 className="preference__label">Increase</h6>
+              </button>
+              <input
+                type="radio"
+                onChange={this.props.indentWithSpace}
+                aria-label="indentation with space"
+                name="indentation"
+                id="indentation-space"
+                className="preference__radio-button"
+                value="Spaces"
+                checked={!this.props.isTabIndent}
+              />
+              <label htmlFor="indentation-space" className="preference__option">Spaces</label>
+              <input
+                type="radio"
+                onChange={this.props.indentWithTab}
+                aria-label="indentation with tab"
+                name="indentation"
+                id="indentation-tab"
+                className="preference__radio-button"
+                value="Tabs"
+                checked={this.props.isTabIndent}
+              />
+              <label htmlFor="indentation-tab" className="preference__option">Tabs</label>
+            </div>
+            <div className="preference">
+              <h4 className="preference__title">Autosave</h4>
+              <div className="preference__options">
+                <input
+                  type="radio"
+                  onChange={() => this.props.setAutosave(true)}
+                  aria-label="autosave on"
+                  name="autosave"
+                  id="autosave-on"
+                  className="preference__radio-button"
+                  value="On"
+                  checked={this.props.autosave}
+                />
+                <label htmlFor="autosave-on" className="preference__option">On</label>
+                <input
+                  type="radio"
+                  onChange={() => this.props.setAutosave(false)}
+                  aria-label="autosave off"
+                  name="autosave"
+                  id="autosave-off"
+                  className="preference__radio-button"
+                  value="Off"
+                  checked={!this.props.autosave}
+                />
+                <label htmlFor="autosave-off" className="preference__option">Off</label>
+              </div>
+            </div>
+          </TabPanel>
+          <TabPanel>
+            <br></br>
+            <div className="preference">
+              <h4 className="preference__title">Add a p5.js or an external library</h4>
+              <h4 className="preference__sketch1">Add a script library resource field</h4>
```

Comment:
this class name should be more descriptive, matching the purpose of the element.

Reply:
Sure, I will do that.